sw/host/opentitanlib/src/spiflash/flash.rs

```diff
@@ -151,12 +151,19 @@ impl SpiFlash {
         loop {
             // READ_SFDP always takes a 3-byte address followed by a dummy byte regardless of
             // address mode.
-            spi.run_eeprom_transactions(&mut [Transaction::Read(
-                MODE_111
-                    .dummy_cycles(8)
-                    .cmd_addr(SpiFlash::READ_SFDP, 0, AddressMode::Mode3b),
-                &mut buf,
-            )])?;
+            let mut eeprom_transactions: Vec<Transaction> = Vec::new();
```

Comment:
You don't need the explicit type annotation here.  The compiler will infer the type based on `eeprom_transactions.push(...)` below.

Reply:
Done